app/views/users/backup_code_setup/confirm_backup_codes.html.erb

```diff
@@ -0,0 +1,35 @@
+<% title t('titles.backup_codes') %>
+
+<%= render PageHeadingComponent.new.with_content(t('titles.backup_codes')) %>
+
+<p>
+  <%= t('two_factor_authentication.backup_codes.warning_html') %>
+</p>
+
+<p>
+  <%= t('two_factor_authentication.backup_codes.instructions', app_name: APP_NAME) %>
+</p>
+
+<div class="padding-left-0 grid-container">
+  <div class="grid-row usa-button--row">
```

Comment:
I'm not familiar with `usa-button--row` . Is it a design system class?

Reply:
it is not. I was messing around with constraining the width on desktop with work on responsiveness to follow.